tests/python_tests/helpers/param_config.py

```diff
@@ -402,3 +478,13 @@ def calculate_edgecase_dest_indices(
             combinations.extend([(dest_sync, max_index)])
 
     return combinations
+
+
+def get_max_dst_index(
+    edgecase_dest_indices: List[Tuple[DestSync, int]]
+) -> Optional[int]:
+    """
+    Get the max destination index from the list of edgecase destination indices.
+    """
+    max_index = max(index for _, index in edgecase_dest_indices)
```

Comment:
```suggestion
    max_index = max((index for _, index in edgecase_dest_indices), default=0)
```
If `edgecase_dest_indices` is empty, simple `max(...)` will raise a `ValueError`. This change will save you from troubles.